### PR TITLE
[Backport 2.6] Update cis-operator imagePullPolicy

### PR DESCRIPTION
**Pull request by doflamingo721**
Updated the image pull policy to `IfNotPresent` so that if the image already exists it won't be pulled from the registry.

Depends on: https://github.com/rancher/cis-operator/pull/171
Adds:
````
cis-benchmark: 2.1.1-rc1
cis-benchmark-crd: 2.1.1-rc1
````
Linked issue: https://github.com/rancher/rancher/issues/39685
SURE: 5093

### CODE REVIEW

File: packages/rancher-cis-benchmark/charts/values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.9
+    tag: v1.0.10-rc1
```
Comment (prachidamle) [inline packages/rancher-cis-benchmark/charts/values.yaml]: can be updated to v1.0.10

**Review by prachidamle — COMMENTED** [review]
Changes lgtm, needs to be rebased to resolve conflict

File: packages/rancher-cis-benchmark/charts/values.yaml (hunk 1 above)
Comment (doflamingo721) [reply packages/rancher-cis-benchmark/charts/values.yaml]: I have updated the cis-operator tag to the latest which is 1.0.11-rc1

**Review by prachidamle — APPROVED** [review]
lgtm

**Pull request merged**